### PR TITLE
Fixes #4595 - Corrected a server name in an example

### LINKED ISSUE #4595: Typo in Example
in one example `Server01` shoukd be replaced with `Server01`.

Would have send `PR`, but the page is avalaible for each version

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2ee831d7-2664-6640-a75e-6b46828c2db9
* Version Independent ID: 064d26b8-6e70-943f-2a84-b0434b100b6c
* Content: [about_Remote_Disconnected_Sessions](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_remote_disconnected_sessions?view=powershell-5.0#feedback)
* Content Source: [reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Acrolinx scores
Original version: 89
Updated version: 93

- Fixes [AB#1576932](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1576932)
- Fixes #4595
- State and Availability section, fixed incorrect server name in example (Server01 > Server30)
- Style, paragraph reflows, and minor copyedits

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 1]
```diff
@@ -112,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 `New-PSSession` or `Get-PSSession` cmdlets to `Disconnect-PSSession`.
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
 PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```
Comment (bobbytreed) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Probably can remove the PS prompt here as well

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 2]
```diff
@@ -113,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
-PS C:\> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```
Comment (bobbytreed) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Remove PS prompt if not needed

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 3]
```diff
@@ -111,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
-PS C:\> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```
Comment (bobbytreed) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Remove PS prompt if not needed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 4]
```diff
@@ -113,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
-PS C:\> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+```
```
Comment (bobbytreed) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Remove PS prompt if not needed

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 5]
```diff
@@ -111,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
-PS C:\> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```
Comment (bobbytreed) [inline reference/6/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Remove PS prompt if not needed

File: reference/7/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md [hunk 6]
```diff
@@ -111,59 +112,67 @@ The first command creates a session to the Server01 computer. The session
...
 
 The following command disconnects the PSSession to the Server01 computer.
-Notice that the value of the State property is Disconnected and the
-Availability is None.
+Notice that the value of the **State** property is **Disconnected** and the
+**Availability** is **None**.
 
 ```powershell
-PS C:\> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
+PS> Get-PSSession -ComputerName Server01 | Disconnect-PSSession
```
Comment (bobbytreed) [inline reference/7/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Remove PS prompt if not needed

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 5 above)
Comment (davidsmatlak) [reply reference/6/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

File: reference/7/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md (hunk 6 above)
Comment (davidsmatlak) [reply reference/7/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md]: Resolved.

**Pull request merged**